docs/ui-coverage/guides/ignore-elements.mdx

```diff
@@ -0,0 +1,58 @@
+---
+sidebar_label: Ignore elements
+title: 'Ignore elements | Cypress UI Coverage Documentation'
+description: 'Learn how to exclude irrelevant elements from your UI Coverage reports to focus on meaningful insights.'
+sidebar_position: 50
+---
+
+# Ignore elements
+
+Not all elements in your application are relevant to your test coverage. Cypress UI Coverage allows you to exclude specific elements from coverage reports, helping you focus on meaningful insights and avoid unnecessary noise. This guide explains why and how to ignore elements in your UI Coverage reports.
+
+## Why ignore elements?
+
+Ignoring elements can be beneficial in the following scenarios:
+
+- **Transitional Elements**: Exclude elements that are still in a loading state or undergoing changes, as they may not represent the final state of the element.
+- **Third-Party Widgets**: Exclude elements controlled by external libraries or third-party integrations.
+
+By ignoring irrelevant elements, you can maintain clean and actionable coverage metrics.
+
+## Identify Elements to Ignore
+
+After recording your tests to Cypress Cloud, review the UI Coverage reports:
+
+1. Navigate to the **UI Coverage** tab in your test run.
+1. Look for elements that consistently appear but don't require testing.
+1. Note down the selectors, attributes, or patterns for these elements.
+
+## Configure Ignored Elements
+
+[Element Filters](/ui-coverage/configuration/elementfilters) in the **App Quality** configuration are used to exclude elements based on their selectors or attributes. To add or modify the configuration for your project, navigate to the **App Quality** tab in your project settings and add an **elementFilters** configuration.
+
+```json
+{
+  "elementFilters": [
+    {
+      "selector": "[data-pendo*]",
```

Comment:
`data-pendo` is often used for analytics so it actually makes for a good identifier I'd say

Reply:
Oh, I was interpretting this previous comment as you saying this is a *good* example, now I see you're recommending a change.